content/en/core/releases/feature_releases.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Feature Releases"
+linkTitle: "Feature Releases"
+weight: 2
+no_list: true
+description: >
+    Feature Releases for the CHT Core Framework
+relatedContent: >
+    core/releases
+    contribute#are-you-a-partner-wondering-how-issues-are-prioritized
+    
+---
+
+The Product Team has two dedicated sub-teams (Allies and Care Teams) who try to deliver the most impactful features at a pace that is faster than our regular release cycle. To this end, select partners working directly with the Product Team can access what are called "Feature Releases". 
+
+## Release names
+
+Each Feature Release will be a production-ready branch of the most recent release of the CHT. The release will have undergone applicable discretionary testing. You can easily identify a Feature Release by its version which follows the pattern of `LATEST_RELEASE-FR-FEATURE`. For example, if the latest release is `3.10.1` then the Feature Release for speedier upgrades would be `3.10.1-FR-speedier-upgrades`. When this feature is found to be of use, it would be automatically included in `3.11.0`, the next minor release after `3.10.1`.
```

Comment:
Some suggestions:
- Italicize the first use of _Feature Release_, as [per style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#italics-to-define-or-introduce-new-terms) ([and APA](https://apastyle.apa.org/style-grammar-guidelines/italics-quotations/italics))
- It's not clear what "applicable discretionary testing" means. It sounds like a fancy way of saying we'll do the testing we think is necessary... can we find a simpler way to say that, or specify what testing will be done? I've made a suggestion with simpler language, feel free to improve.
- I thought the pattern for version could be clearer by showing the variable parts.
- As discussed on call, we don't know by which version a feature will be ready to be introduced. I've simplified the text, but open to other suggestions.

```suggestion
Each _Feature Release_ is a production-ready release based on a recent release of the CHT that has gone through the necessary testing. You can easily identify a Feature Release by its version, which follows the pattern of `{BASE-RELEASE-NUMBER}-FR-{FEATURE-NAME}`. For example, if the latest release is `3.10.1` then the Feature Release for speedier upgrades would be `3.10.1-FR-speedier-upgrades`. When this feature is found to be successful and ready for wider distribution it will be included in the next release.
```

Reply:
Yeah - looks good - thanks for the input!  

That said @craig-landry really liked the phrase "discretionary testing".  Are you OK with @abbyad's suggestion to remove it?

Reply:
Yep, I agree that "discretionary testing" is another way of saying "we're going to test it as we see fit"
I liked "discretionary" since it's a short way to say "we'll do the testing we think is necessary". I'm cool with alternatives, I just haven't heard anything I liked better than that. I'm probably fine with whatever other suggestions people have though.

To the point of a suggestion, I don't see it here as part of this suggestion though; instead it looks to be omitted. Was that the intention or am I missing it somewhere else?

Reply:
chatted w/ @craig-landry  - we're good to go as is - but acknowledging we might revisit to add explicit wording around how much we test.